windows-gaming-driver/src/driver/clipboard.rs

```diff
@@ -184,9 +180,9 @@ impl X11Clipboard {
             self.connection.flush();
 
             Ok(())
-        }).map_err(|()| unimplemented!())).then(|_| Ok(())));
+        });
 
-        ClipboardContext { listener, lost_recv, query_recv, data_recv, resp_send }
+        Box::new(xcb_listener.join(responder.map_err(|()| unimplemented!())).then(|_| Ok(())))
```

Comment:
I mean
Does this even matter? We're receiving from an unbounded stream. This can't ever fail.